contracts-periphery/foundry.toml

```diff
@@ -1,5 +1,5 @@
 [default]
-remappings = ['test/=test/']
+verbosity = 3
 [ci]
```

Comment:
Nit, let's get a blank line between profiles

Reply:
Should be fixed now